### DISCUSSION TITLE
Get started with nuxt 3

### DISCUSSION THREAD

**Discussion by hermesalvesbr** [apostrophecms/apostrophe#3662] [Q&A]
Hi there,

I'm looking for a tutorial or course, for learning Apostrophe to make a website with dynamic menu and logged user. 

Where may i find this tutorial? Does anyone want to charge to do this tutorial?

Kind regards.

**Comment by boutell** [comment]
Hi Hermes,

Nuxt might be overkill if your goal is a website, as opposed to an
application. Apostrophe is well-positioned to create websites already. Vue
can be used to help enhance the experience but it is not mandatory (you can
use any JS framework of your choice).

On Wed, Feb 16, 2022 at 3:18 PM Hermes Alves ***@***.***>
wrote:

> Hi there,
>
> I'm looking for a tutorial or course, for learning Apostrophe to make a
> website with dynamic menu and logged user.
>
> Where may i find this tutorial? Does anyone want to charge to do this
> tutorial?
>
> Kind regards.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/apostrophecms/apostrophe/discussions/3662>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAH27M2ODUR3Y6VV45Y3DLU3QBCFANCNFSM5OSVPHYA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 

THOMAS BOUTELL | CHIEF TECHNOLOGY OFFICER
APOSTROPHECMS | apostrophecms.com | he/him/his
